cg_hermes/constants/tags.py

```diff
@@ -262,7 +282,7 @@ def name(cls) -> str:
     def description(self) -> str:
         descriptions: dict[QCTags, str] = {
             self.PED_CHECK: "Results from pedigree validation",
-            self.QC_METRICS: "QC metrics from analysis",
+            self.QC_METRICS: "QC metrics from analysis",  # metrics_deliverables file
```

Comment:
Not exclusively, we also assign this tag to the multiqc files

```suggestion
            self.QC_METRICS: "QC metrics from analysis", 
```